Remove config/manager/kustomization.yaml from .gitignore

kajinamit opened the pull request:
This file is required to build a bundle image. The file is actually maintained in the repository.

gibizer reviewed (approved):
/lgtm Bugfix release. Only contains the lib version bump.

kajinamit committed:
Remove config/manager/kustomization.yaml from .gitignore

This file is required to build a bundle image. The file is actually
maintained in the repository.

kajinamit commented:
This is intentional to avoid pulling changes by `make bundle`.
